common/kvstore/KeyValueStore-emscripten.cc

```diff
@@ -60,17 +60,13 @@ void OwnedKeyValueStore::refreshMainTransaction() {
     throw_mdb_error("creating databases isn't supported on emscripten"sv, 0);
 }
 
-u4 OwnedKeyValueStore::sessionId() const {
-    return 0;
-}
-
 unique_ptr<KeyValueStore> OwnedKeyValueStore::bestEffortCommit(spdlog::logger &logger,
                                                                unique_ptr<OwnedKeyValueStore> k) {
-    throw_mdb_error("creating databases isn't supported on emscripten"sv, 0);
+    return nullptr;
```

Comment:
why change in behavior?

Reply:
It makes it benign to pass a nullptr kvstore to these functions, which simplifies code that converts between kvstore and owned kvstore.

If you disagree I can add explicit null checks back in.